client/python/README.md

```diff
@@ -1,21 +1,157 @@
-To run this project, first ensure that ModelDB Server is up and running. See instructions [here](../../server).
+# Introduction
 
-Run:
-```
-./build_client.sh
-``` 
-Put the python client on your PYTHONPATH
-```
-export PYTHONPATH=path_to_modedb_dir/client/python:$PYTHONPATH
+This is the Python client for ModelDB, which includes ModelDB's Light API and the [scikit-learn](http://scikit-learn.org/stable) client.
```

Comment:
[MAJOR] I think we should make the Light API and sklearn into two separate READMEs. These are two v. diff code paths that support diff functionality. Having them together will confuse people.

So let's make a separate read me for LIGHT. You can say that it's written in Python of course.